packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -251,7 +251,8 @@ export const makeRoundsManagerKit = defineDurableExoClassKit(
           answeredInRound: 0n,
         });
         rounds.init(roundId, round);
-        latestRoundPublisher.publish({
+        // XXX how to handle errors here?
+        E(latestRoundPublisher).write({
```

Comment:
`void` seems reasonable.

That is: rely on whatever unhandled rejection policy is used throughout the platform. Current state of the art seems to make them reasonably visible to node operators: #4939 / #4471. See also [logging issues](https://github.com/Agoric/agoric-sdk/labels/logging).

As to RPC clients: if writing latest round info fails, is writing an error message likely to succeed?

And if it's a transient problem, we can expect it to self-heal, since rounds happen on a regular cadence.